test/cql-pytest/test_using_timeout.py

```diff
@@ -1,6 +1,7 @@
 # Copyright 2020-present ScyllaDB
 #
 # SPDX-License-Identifier: AGPL-3.0-or-later
+import re
```

Comment:
Is this change connected with this PR?

Reply:
No, it's not related but pycharm was complaining about the missing import in this file, so decided to add this import.

Reply:
It seems this code has an unused function calling re.compile(). The "import re" wasn't needed because it's unused - I guess you used an over-zealous linter which complained. The better solution is just to remove the unused function. In any case it's unrelated to this PR.

Reply:
https://github.com/scylladb/scylladb/pull/17801

Reply:
I didn't use any additional linters. There were changes related to my scope with gather(). So I have no idea why I opened this file since the only change is adding this import. But I agree it's better to remove unused code.